digitalocean_plugin/security.py

```diff
@@ -14,21 +14,138 @@
 # limitations under the License.
 
 import os
```

Comment:
There's a convention we try to keep which is separating the imports into three classes ```
# Built-in Imports
...
# Third party Imports
...
# Cloudify Imports
```
See: https://github.com/cloudify-cosmo/cloudify-aws-plugin/blob/master/ec2/tests/test_ec2_workflow.py#L16